corehq/motech/repeaters/dbaccessors.py

```diff
@@ -153,6 +154,34 @@ def _get_repeater_ids_by_domain(domain):
     return [result['id'] for result in results]
 
 
+def get_repeat_records_for_ids(doc_ids):
+    from .models import RepeatRecord
+    return [RepeatRecord.wrap(doc['doc']) for doc in get_docs(RepeatRecord.get_db(), doc_ids)]
+
+
+def get_repeat_record_ids(due_before, limit=None):
+    """
+    Fetch repeat record ids only
+    :param due_before: only return repeat records that are due now or sooner
+    :param limit: limit fetched record ids, no limit by default
+    :return: a list of repeat record ids
+    """
+    from .models import RepeatRecord
+    json_now = json_format_datetime(due_before)
+
+    view_kwargs = {
+        'reduce': False,
+        'startkey': [None],
+        'endkey': [None, json_now, {}],
+        'include_docs': False,
+    }
+
+    if limit is not None:
+        view_kwargs['limit'] = limit
```

Comment:
I'd avoid prematurely parameterizing functions that are only used once, because it introduces forks in the code like this even though `if` statement is only ever evaluated one way.  One exception is if the parameter makes more logical sense in the calling code for some reason (differing abstraction levels), though in that case I'd still give preference to a required argument over an optional one.